template-only-docs/Deployment.md

```diff
@@ -0,0 +1,43 @@
+# Deployment to cloud environments
+
+## Requirements
+
+- External service access: The application must be able to make network calls to the public internet.
+- AWS Cognito: The application comes with AWS Cognito enabled by default, so AWS Cognito must be set up before deploying the application.
+- Custom domain name with HTTPS support: AWS Cognito requires HTTPS for callback urls to function.
+- Access to write to temporary directory: Rails needs to be able to write out temporary files.
+- Environment variables: You must provide the application with the environment variables listed in [local.env.example](/app-rails/local.env.example).
```

Comment:
Do you want to say more about this specifically for cloud deployments? I know we're setting them as aws secrets and importing them to the ecs containers via the task definition, but if I'm using this repo and not using the infra template, that context would be really helpful.